algrtm/src/components/OfferSection.js

```diff
@@ -0,0 +1,220 @@
+              <div className="flex items-center gap-4  w-full pb-4 justify-between">
+                <div>Tailored Data Quality & Model Training Pipelines</div>
+
+                <div className="w-5 h-5">
+                  <Icon
+                    src={
+                      textVisibility[3]
+                        ? "/arrow-up-icon.svg"
+                        : "/arrow-down-icon.svg"
+                    }
+                    alt="Arrow icon"
+                    onClick={() => handleOffer(3)}
+                    width={20}
+                  />
+                </div>
+              </div>
+
+              {textVisibility[3] && (
+                <div className="max-w-[25.75rem]">
+                  <p className="text-neutralLight-neutral20 p2 leading-[1.64rem] pb-4">
+                    text 3
+                  </p>
+                </div>
+              )}
+            </div>
+
+            <div className="flex flex-col border-b-[1px] border-neutralLight-neutral90">
+              <div className="flex items-center gap-4 self-stretch w-full pb-4 justify-between">
+                <div>End-To-End Machine Learning Development</div>
+
+                <div className="!w-5 !h-5">
+                  <Icon
+                    src={
+                      textVisibility[4]
+                        ? "/arrow-up-icon.svg"
+                        : "/arrow-down-icon.svg"
+                    }
+                    alt="Arrow icon"
+                    onClick={() => handleOffer(4)}
+                  />
+                </div>
+              </div>
+
+              {textVisibility[4] && (
+                <div className="max-w-[25.75rem]">
+                  <p className="text-neutralLight-neutral20 p2 leading-[1.64rem] pb-4">
+                    TEXT 4
+                  </p>
+                </div>
+              )}
+            </div>
+
+            <div className="flex flex-col border-b-[1px] border-neutralLight-neutral90">
+              <div className="flex items-center gap-4 self-stretch w-full pb-4 justify-between">
+                <div>Performance Enhancement and Scaling</div>
+
+                <div className="!w-5 !h-5">
+                  <Icon
+                    src={
+                      textVisibility[5]
+                        ? "/arrow-up-icon.svg"
+                        : "/arrow-down-icon.svg"
+                    }
+                    alt="Arrow icon"
+                    onClick={() => handleOffer(5)}
+                  />
+                </div>
+              </div>
+
+              {textVisibility[5] && (
+                <div className="max-w-[25.75rem]">
+                  <p className="text-neutralLight-neutral20 p2 leading-[1.64rem] pb-4">
+                    TEXT 5
+                  </p>
+                </div>
+              )}
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+};
+
+export default OfferSection;
```

Comment:
Instead of manually using textVisibility[0], textVisibility[1], textVisibility[2], etc., the easiest way is to create an array of objects where we specify the `title` and `description` of each section and store its `visibility status`. Then using `.map`, we can iterate over each element of the array and dynamically create this component

In addition, ideally, changing the visibility status should be possible by clicking on the entire section, not just the icon, which is sometimes hard to hit (especially on a mobile devices). So the onClick event can be added directly to parent div:
```jsx
<div
  className='flex items-center gap-4 self-stretch w-full pb-4 justify-between'
  onClick={() => handleOffer(1)}>
  <div>Machine Learning Feasibility Study</div>

  <div>
    <Icon
      src={
        textVisibility[1]
          ? "/arrow-up-icon.svg"
          : "/arrow-down-icon.svg"
      }
      alt='Arrow icon'
      width={20}
    />
  </div>
</div>
```
Along with this change, we should also add some `aria` attributes to improve accessibility